examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/SuccessCallback.java

```diff
@@ -17,6 +17,18 @@
  */
 package com.chip.casting;
 
-public interface SuccessCallback<R> {
-  void handle(R response);
+import android.util.Log;
+
+public abstract class SuccessCallback<R> {
+  private static final String TAG = SuccessCallback.class.getSimpleName();
+
+  public abstract void handle(R response);
+
+  public void handleInternal(R response) {
+    try {
+      handle(response);
+    } catch (Exception e) {
```

Comment:
if the purpose is avoiding a crash, then should this be catching Throwable rather than Exception?

Reply:
Done